api/model/UserData.v1.yaml

```diff
@@ -14,11 +18,8 @@ properties:
     example: '1990-01-01'
   creationDate:
     type: string
```

Comment:
esiste un tipo date?

Reply:
in effetti si. https://swagger.io/docs/specification/data-models/data-types/

è un format del type string. provvedo a correggere!